server/metrics_server.go

```diff
@@ -0,0 +1,34 @@
+package server
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/log"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
```

Comment:
third-party library import - move above the kiali imports into its own section

Reply:
we can fix this later - trivial - but, convention is "standard libs, blank space, third party libs, blank space, kiali libs" -- the third party lib should be up above the kiali libs.